apisix/cli/ops.lua

```diff
@@ -406,6 +408,21 @@ local function start(env, ...)
         end
     end
 
+    local parser = argparse()
+    parser:argument("_", "Placeholder")
+    parser:option("-c --config", "location of customized config.yaml")
+    local args = parser:parse()
+    local customized_yaml = args["config"]
+
+    profile.apisix_home = env.apisix_home .. "/"
+    local local_conf_path = profile:yaml_path("config")
+
+    if customized_yaml then
+        execute("mv " .. local_conf_path .. " " .. local_conf_path .. ".bak")
+        execute("ln " .. customized_yaml .. " " .. local_conf_path)
```

Comment:
Use `execute_cmd` in `apisix.cli.utils` .

Reply:
Fixed. Thanks!